tests/framework/pvc.go

```diff
@@ -46,6 +47,11 @@ func (f *Framework) FindPVC(pvcName string) (*k8sv1.PersistentVolumeClaim, error
 	return utils.FindPVC(f.K8sClient, f.Namespace.Name, pvcName)
 }
 
+// WaitPVCDeletedByUID is a wrapper around utils.WaitPVCDeletedByUID
+func (f *Framework) WaitPVCDeletedByUID(pvcSpec *k8sv1.PersistentVolumeClaim, timeout time.Duration) (bool, error) {
```

Comment:
This calls uitls.WaitPVCDeleted, the UID is not important here either.

Reply:
Same comment as above.

Reply:
Agreed